.github/workflows/ci.yml

```diff
@@ -14,28 +14,34 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.12"]
-        toxenv: [quality, docs, pii_check, django42, django52]
+        task: ["quality:check", "test:test"]
     steps:
       - uses: actions/checkout@v4
       - name: setup python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: |
+            3.11
+            3.12
 
-      - name: Install pip
-        run: pip install -r requirements/pip.txt
+      - name: Cache package
```

Comment:
@xitij2000 Why are we using Hatch here? I don't have an opinion on this. I am mostly curious on the choice. I have been using uv recently and noticed it also [has support for Hatch as a backend](https://docs.astral.sh/uv/reference/cli/#uv-init--build-backend).

Reply:
I'm using hatch since it's something I'm familiar with, and it's a project under pypa and support multiple environments making it feasible to replace tox, and eventually even pylint etc.